interpreter/CMakeLists.txt

```diff
@@ -449,7 +452,7 @@ if (builtin_cling)
   # LLVM doesn't really give us a API to get this with an in-source build
   # so we just use the normal way of doing this and read the llvm directory
   # compilation properties.
-  get_directory_property(LLVM_DEFS DIRECTORY llvm/src COMPILE_DEFINITIONS)
+  set(LLVM_DEFS "_GNU_SOURCE;__STDC_CONSTANT_MACROS;__STDC_FORMAT_MACROS;__STDC_LIMIT_MACROS")
```

Comment:
`_GNU_SOURCE` seems to be defined only if `LLVM_USING_GLIBC` is set. That's probably what breaks osx.

Reply:
```suggestion
  if( LLVM_USING_GLIBC )
    set(LLVM_DEFS "_GNU_SOURCE;__STDC_CONSTANT_MACROS;__STDC_FORMAT_MACROS;__STDC_LIMIT_MACROS")
  endif()
```